sample.py

```diff
@@ -0,0 +1,60 @@
+# This function adds two numbers
+def add(x, y):
+    return x + y
+
+
+# This function subtracts two numbers
+def subtract(x, y):
+    return x - y
+
+
+# This function multiplies two numbers
+def multiply(x, y):
+    return x * y
+
+
+# This function divides two numbers
+def divide(x, y):
+    if y == 0:
+        raise ValueError("Cannot divide by zero")
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider more descriptive error message</b></div>
<div id="fix">

Consider adding more descriptive error message in `ValueError` that includes the actual values causing the error. This would help with debugging.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
        raise ValueError(f"Cannot divide by zero (y={y})")
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/15#issuecomment-2535490133>#77523e</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged